go.mod

```diff
@@ -0,0 +1,110 @@
+module github.com/njuettner/external-dns
```

Comment:
```suggestion
module github.com/kubernetes-incubator/external-dns
```

I was only able to run the tests after changing the module name. Otherwise the following errors popped up:

```
# github.com/njuettner/external-dns/controller
controller/controller_test.go:24:2: use of internal package github.com/kubernetes-incubator/external-dns/internal/testutils not allowed
FAIL    github.com/njuettner/external-dns/controller [setup failed]
# github.com/njuettner/external-dns/plan
plan/plan_test.go:23:2: use of internal package github.com/kubernetes-incubator/external-dns/internal/testutils not allowed
FAIL    github.com/njuettner/external-dns/plan [setup failed]
# github.com/njuettner/external-dns/provider
provider/aws_sd_test.go:29:2: use of internal package github.com/kubernetes-incubator/external-dns/internal/testutils not allowed
FAIL    github.com/njuettner/external-dns/provider [setup failed]
# github.com/njuettner/external-dns/registry
registry/aws_sd_registry_test.go:23:2: use of internal package github.com/kubernetes-incubator/external-dns/internal/testutils not allowed
FAIL    github.com/njuettner/external-dns/registry [setup failed]
# github.com/njuettner/external-dns/source
source/dedup_source_test.go:23:2: use of internal package github.com/kubernetes-incubator/external-dns/internal/testutils not allowed
FAIL    github.com/njuettner/external-dns/source [setup failed]
```